cluster/storage/src/main/java/com/linkedin/openhouse/cluster/storage/adls/AdlsStorageClient.java

```diff
@@ -0,0 +1,68 @@
+package com.linkedin.openhouse.cluster.storage.adls;
+
+import com.azure.storage.file.datalake.DataLakeFileClient;
+import com.linkedin.openhouse.cluster.storage.BaseStorageClient;
+import com.linkedin.openhouse.cluster.storage.StorageType;
+import com.linkedin.openhouse.cluster.storage.configs.StorageProperties;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.HashMap;
+import java.util.Map;
+import javax.annotation.PostConstruct;
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Component;
+
+/**
+ * AdlsStorageClient is an implementation of the StorageClient interface for Azure Data Lake
+ * Storage. It uses the {@link DataLakeFileClient} to interact with Data Lake Storage.
+ */
+@Slf4j
+@Lazy
+@Component
+public class AdlsStorageClient extends BaseStorageClient<DataLakeFileClient> {
+
+  private static final StorageType.Type ADLS_TYPE = StorageType.ADLS;
+
+  @Autowired private StorageProperties storageProperties;
+
+  @Getter private Map properties;
```

Comment:
lets remove this getter, the properties should be available through `AdlsStorage.getProperties()`

Reply:
Sounds good